docs/blog/newsletter-september-2021.rst

```diff
@@ -0,0 +1,125 @@
+- Rewrite long paragraphs to use lists. Condense the most critical points to eye-catching items in a list and make the text more skimmable. Write instructions in individually numbered steps.
+- Look for opportunities to use formatting to add meaning to the text. Use special icons or different background shading to delineate notes and warnings. Put code snippets, quotations, and math equations in their own indented blocks. Use bold font weight to make keywords stand out. Left-justify the first paragraph for each topic and block-indent subsequent paragraphs on the same topic.
+
+-----------------------------------
+Where are all the junior positions?
+-----------------------------------
+
+An interesting topic circulating in the WTD community is the scarcity of entry-level technical writing jobs. Nowadays, most employers seem to be looking for writers with at least five years of experience. But why is that the case? Perhaps because teams need the help of a seasoned writer to help them grow, but also because, in most situations, an experienced writer provides a greater return on investment than a junior writer. If a team is overloaded, they might feel unable to give someone less experienced the training, guidance, and mentorship they need.
+
+So, how can junior writers find jobs when it feels like the odds are stacked against them? One suggestion is to pick a niche and dig into it. By becoming proficient in a specialized subject, new writers will be better equipped to compete in the job market - as other candidates for entry-level tech writing roles can be engineers or support engineers wanting to transfer into writing-oriented roles.
+
+Another approach is to tap into your professional network. In a saturated industry, knowing someone who can help you get more experience on your resume can help hugely. After gaining some experience, seeking new positions becomes much easier - though that doesn't mean it won't be challenging still.
+
+The best advice for those interested in tech writing, regardless of the industry, is to establish domain knowledge in a subject, leverage professional networks, create a portfolio of work, and explore opportunities that allow for continuous learning and skill building to achieve a fulfilling career. For more information on starting a tech writer career, check out `this article on Career Explorer <https://www.careerexplorer.com/careers/technical-writer/how-to-become/>`__.
+
+--------------------------------------
+The basics you need to document an API
+--------------------------------------
+
+A discussion this month around demystifying API documentation, we asked: what's the minimum that you need to know to document an API? Here's what the community thought:
```

Comment:
Should this sentence say "In a discussion this month" or something like that instead?

Reply:
yes, fixed, thanks!